docs/source/models.rst

```diff
@@ -1304,129 +1304,105 @@ For more info, see:
 Deploy a ``python_function`` model on Microsoft Azure ML
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The :py:mod:`mlflow.azureml` module can package ``python_function`` models into Azure ML container images and deploy them as a webservice. Models can be deployed to Azure Kubernetes Service (AKS) and the Azure Container Instances (ACI)
-platform for real-time serving. The resulting Azure ML ContainerImage contains a web server that
-accepts the following data formats as input:
+The MLflow plugin `azureml-mlflow <https://pypi.org/project/azureml-mlflow/>`_ can deploy models to Azure ML, either to Azure Kubernetes Service (AKS) or Azure Container Instances (ACI) for real-time serving. 
+
+The resulting container image contains a web server that accepts the following data formats as input:
 
 * JSON-serialized pandas DataFrames in the ``split`` orientation. For example, ``data = pandas_df.to_json(orient='split')``. This format is specified using a ``Content-Type`` request header value of ``application/json``.
 
-* :py:func:`mlflow.azureml.deploy` registers an MLflow Model with an existing Azure ML workspace, builds an Azure ML container image and deploys the model to AKS and ACI. The `Azure ML SDK`_ is required in order to use this function. *The Azure ML SDK requires Python 3. It cannot be installed with earlier versions of Python.*
+Deployments can be generated using both the Python API or MLflow CLI. On both cases, a ``JSON`` configuration file has to be specified with the details of the deployment you want to achieve. If not indicated, then a default deployment is done using Azure Container Instances (ACI). You can check the full specification at `Deployment configuration schema <https://docs.microsoft.com/en-us/azure/machine-learning/reference-azure-machine-learning-cli#deployment-configuration-schema>`_. The deployment configuration file looks as follows:
```

Comment:
```suggestion
Deployments can be generated using both the Python API or MLflow CLI. In both cases, a ``JSON`` configuration file has to be specified with the details of the deployment you want to achieve. If not indicated, then a default deployment is done using Azure Container Instances (ACI). You can view the full specification at `Deployment configuration schema <https://docs.microsoft.com/en-us/azure/machine-learning/reference-azure-machine-learning-cli#deployment-configuration-schema>`_. The deployment configuration file looks as follows:
```

A couple questions here:

1. The user doesn't actually have to pass a file, right? They can pass a dictionary?
2. We say that some JSON "has to be specified", but then we discuss a case where defaults are used when the JSON is not specified. Can we clarify?
3. The following JSON code blocks are just examples containing a subset of the available configuration options, right? Can we clarify that?

Reply:
1. Yes, the user has to pass a file right now :(. In-place dictionaries are not supported since this is a wrapper on the client.
2. I got you point. I wanted to stress out the idea that if you don't indicate a JSON deployment file, one will be generated for you with some defaults. So yes, it is optional in the client, but no, the service needs one. I agree I had to make that more clear. Addressed in next commit.
3. I think It is stated on "You can view the full specification at [Deployment configuration schema](https://docs.microsoft.com/en-us/azure/machine-learning/reference-azure-machine-learning-cli#deployment-configuration-schema). But I can make it more clear.